# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 984db15bc9e2
# Full Hash: 984db15bc9e2b5baa20bdc86baaa1165c381d326
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 11: Migrate ShowPersistentNotification to use IPC r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227696
# ==============================================================================

diff -r 622a66cdac2e -r 984db15bc9e2 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Thu Nov 07 17:32:07 2024 +0000
+++ b/dom/ipc/ContentParent.h	Thu Nov 07 17:32:07 2024 +0000
@@ -1011,9 +1011,6 @@
   mozilla::ipc::IPCResult RecvOpenNotificationSettings(
       nsIPrincipal* aPrincipal);
 
-  mozilla::ipc::IPCResult RecvNotificationEvent(
-      const nsAString& aType, const NotificationEventData& aData);
-
   mozilla::ipc::IPCResult RecvLoadURIExternal(
       nsIURI* uri, nsIPrincipal* triggeringPrincipal,
       nsIPrincipal* redirectPrincipal,